naples/routes/utils.py

```diff
@@ -34,3 +34,35 @@ def get_user_data(user: m.User) -> s.User:
     log(log.INFO, f"User data: {user_data}")
 
     return s.User.model_validate(user_data)
+
+
+def get_product_data(product: m.Product) -> s.ProductOut:
```

Comment:
Why can't we use `s.ProductOut.model_validate(product)` instead of this custom function?